lv_port/lv_port_display_espressif.c

```diff
@@ -55,3 +85,77 @@ bool display_port_gpio_is_busy(lv_disp_drv_t *drv)
 
     return device_busy;
 }
+
+void display_interface_send_cmd(lv_disp_drv_t *drv, uint32_t cmd, cmd_width_t cmd_width, void *args, size_t args_len)
+{
+    (void) drv;
+
+    if (display_interface_is_spi(drv)) {
+        disp_wait_for_pending_transactions();
+        display_port_gpio_dc(drv, LV_DISPLAY_DC_CMD_MODE);
+
+        if (CMD_WIDTH_8BITS == cmd_width) {
+            uint8_t cmd_8bits = (uint8_t) cmd & 0xFFU;
+            disp_spi_send_data(&cmd_8bits, 1);
+        }
+        else if (CMD_WIDTH_16BITS == cmd_width) {
+            /* Send 16bits cmd */
+        }
+        else {
+            /* Invalid cmd size */
+            assert(0);
+        }
+
+        if (args != CMD_WITHOUT_ARGS) {
+            display_port_gpio_dc(drv, LV_DISPLAY_DC_DATA_MODE);
+            disp_spi_send_data(args, args_len);
+        }
+    }
+
+    if (display_interface_is_i2c(drv)) {
+        uint8_t *data = (uint8_t *) args;
+        lvgl_i2c_write(OLED_I2C_PORT, OLED_I2C_ADDRESS, cmd, data, args_len);
+    }
+}
+
+void display_interface_send_data(lv_disp_drv_t *drv, void *data, size_t len)
```

Comment:
Finally the 
```c
void display_interface_send_data(lv_disp_drv_t *drv, lv_area_t * area,  void *data, size_t len)
```
prototype we discussed earlier won't be used here?

Reply:
To be honest I don't know, the way the drivers currently send the area information is very different to each other. I'm finishing porting a couple of drivers to use this interface to look how similar/different they end up being.

Reply:
Okay!